DuplicateAssemblyScanner/DuplicateAssemblyScanner/Log.cs

```diff
@@ -67,6 +72,16 @@ private enum LogLevel {
             Error,
         }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the log level should be logged.
+        /// </summary>
+        public static bool ShowLogLevel { get; set; } = false;
```

Comment:
Just a hint: the `bool` properties and fields are automatically initialized by the runtime to `false`.
So generally, setting the initial value to `false` is not necessary here.